features/base64encodedecode.yml

```
@@ -0,0 +1,7 @@
+name: Base64 encoding and decoding
+description: Utility functions for encoding and decoding strings to and from base 64.
```

We've got the room here, so we can name both "entrypoints" to this feature and make it a complete sentence.

```suggestion
description: The `atob()` and `btoa()` global functions encode and decode strings to and from base 64, respectively.
```